template.yml

```diff
@@ -55,7 +55,7 @@ Resources:
     Properties:
       CodeUri: ./target/MSKToS3Lambda-1.0-SNAPSHOT.jar
       Handler: com.amazonaws.kafka.samples.HandlerMSK
-      Runtime: java8
+      Runtime: java8.al2
```

Comment:
wouldn't this work with Kafka 11 or 17?

Reply:
@awsalialem  this would require changing the deployed java version to 17 as well which would break the existing workshop. i would handle java upgrade in the new workshop